src/app/scopes-dialog/scopes-dialog.component.ts

```
@@ -188,14 +188,7 @@ export class ScopesDialogComponent extends GraphExplorerComponent implements Aft
   public async getNewAccessToken() {
     const selectedScopes = PermissionScopes.filter((scope) => scope.requested && !scope.consented)
       .map((scope) => scope.name);
-
-    try {
-      await acquireNewAccessToken(app, selectedScopes).then((response) => {
-        return;
-      });
-    } catch (error) {
-      throw error;
```

Was this exception handled anywhere? If so, please remove that handler since this is no longer expected.

---

No the exception was not handled anywhere else.